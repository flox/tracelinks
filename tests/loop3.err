ERROR: loop detected in path: ./tests/loop3 -> ./tests/loop1 -> ./tests/loop2 -> ./tests/loop3
