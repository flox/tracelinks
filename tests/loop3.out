./tests/loop3 -> loop1
  ./tests/loop1 -> loop2
    ./tests/loop2 -> loop3
